projects/ion/src/lib/tab-group/tab-group.component.ts

```diff
@@ -47,9 +54,11 @@ export class TabGroupComponent implements OnInit {
   }
 
   public selectTab(tabSelected: TabInGroup) {
-    this.clearTabs();
-    tabSelected.selected = true;
-    this.selected.emit(tabSelected);
+    if (!tabSelected.selected) {
+      this.clearTabs();
+      tabSelected.selected = true;
```

Comment:
```suggestion
      tabSelected.select();
```

Call method, this way if there is a change in logic the behavior will be replicated here.